v2/azure-arm.yaml

```diff
@@ -1238,6 +1238,32 @@ objectModelConfiguration:
         $export: true
         $supportedFrom: v2.0.0-beta.2
         $importable: false
+    2020-06-01:
+      PrivateDnsZones_VirtualNetworkLink:
+        $exportAs: PrivateDnsZonesVirtualNetworkLink
+        $supportedFrom: v2.0.0
+        $importable: false
+      PrivateDnsZones_A:
+        $exportAs: PrivateDnsZonesRecordsetTypeA
+        $supportedFrom: v2.0.0
```

Comment:
Do we need to support `FromConfig` for some of the A/AAAA record inputs, such as the IP addr?

Reply:
> do we also need to support via raw IP (IP exported from PE, imported into ARecord, via configmap?

A record gets automatigically created in privateDNSZone

Reply:
Hence, we don't require such